layouts/partials/header.html

```diff
@@ -187,9 +188,10 @@
                 <a class="menu subscription" data-action="subscription"
                     data-target="subscription" title="Suscríbete al Podcast">Suscríbete</a>
               {{ end }}
-              <a class="nav-social" target="_blank" href="https://twitter.com/{{ .Site.Params.twitter }}" title="Entre Dev Y Ops en Twitter"><i class="fa fa-twitter"></i></a>
-              <a class="nav-social" target="_blank" href="https://github.com/{{ .Site.Params.github }}" title="Entre Dev Y Ops en GitHub"><i class="fa fa-github"></i></a>
-              <a id="search-button" onclick="$('#search-form').toggle();" title="Buscar"><i class="fa fa-search"></i></a>
+              <a class="nav-social" target="_blank" href="https://twitter.com/{{ .Site.Params.twitter }}" title="Entre Dev y Ops en Twitter"><i class="fab fa-twitter"></i></a>
+              <a class="nav-social" target="_blank" href="https://t.me/{{ .Site.Params.telegram }}" title="Entre Dev y Ops en Telegram"><i class="fab fa-telegram-plane"></i></a>
+              <a class="nav-social" target="_blank" href="https://github.com/{{ .Site.Params.github }}" title="Entre Dev y Ops en GitHub"><i class="fab fa-github"></i></a>
```

Comment:
I think these titles should be using some var, maybe even `.Title`. I might be wrong, though.

Reply:
Yep, I will fix it in other PR.